Sources/App/Migrations/064/UpdateTargetAddType.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright Dave Verwer, Sven A. Schmidt, and other contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Fluent
+
+struct UpdateTargetAddType: AsyncMigration {
+    func prepare(on database: Database) async throws {
+        try await database.schema("targets")
+            .field("type", .json)
```

Comment:
We'll want to make this `.required` at some point but we'll have to have re-analysed all versions and populated the field for that.